share/spice/sound_cloud/sound_cloud.js

```diff
@@ -66,7 +65,9 @@
                     return;
                 }
 
-                var streamURL = '/audio/?u=' + o.stream_url + '?client_id=' + SOUNDCLOUD_CLIENT_ID;
+                var streamURL = '/js/spice/sound_cloud_stream/' + o.stream_url.replace(/https?:\/\//, '');
```

Comment:
The browser was making requests to `https:/` instead of `https://` -- I think NGINX was merging the double `//` causing problems.

I tried using `encodeURIComponent` but then the request failed. This approach managed to work